sam_view.c

```diff
@@ -293,6 +294,7 @@ int main_samview(int argc, char *argv[])
 
     static const struct option lopts[] = {
         SAM_OPT_GLOBAL_OPTIONS('-', 0, 'O', 0, 'T', '@'),
+        {"no-PG", no_argument, NULL, 1},
```

Comment:
It would be good to have a `--no-pg` alias as well for typing ease.

In fact, I wonder whether it would be practical to move this into `sam_global_args` rather than duplicating the code everywhere…